packages/core/src/Utils.ts

```diff
@@ -1,11 +1,25 @@
 import nanoid from 'nanoid';
 
-import { debugAndPrintError } from './Debug';
+import {debugAndPrintError, debugMatchMaking} from './Debug';
+import {
+  createRoomReferences,
+  disposeRoom,
+  driver, handlers,
+  lockRoom, onClientJoinRoom, onClientLeaveRoom,
+  presence,
+  processId,
+  publicAddress,
+  remoteRoomCall, unlockRoom
+} from "./MatchMaker";
```

Comment:
No need to import most of these, since `handleCreateRoom` and related methods should live in the `MatchMaker.ts` file. (No need to add `export` to most of them, too)